server/src/main/java/org/candlepin/model/Content.java

```diff
@@ -200,12 +321,66 @@ public void setId(String id) {
         this.id = id;
     }
 
-    public void setOwner(Owner owner) {
-        this.owner = owner;
+    /**
+     * Retrieves the owners with which this content is associated. If this content is not associated
+     * with any owners, this method returns an empty set.
+     * <p/>
+     * Note that changes made to the set returned by this method will be reflected by this object
+     * and its backing data store.
+     *
+     * @return
+     *  The set of owners with which this content is associated
+     */
+    @XmlTransient
+    @Override
+    public Collection<Owner> getOwners() {
+        return this.owners;
+    }
+
+    /**
+     * Associates this content with the specified owner. If the given owner is already associated
+     * with this content, the request is silently ignored.
+     *
+     * @param owner
+     *  An owner to be associated with this content
+     *
+     * @return
+     *  True if this content was successfully associated with the given owner; false otherwise
+     */
+    public boolean addOwner(Owner owner) {
+        return owner != null ? this.owners.add(owner) : false;
+    }
+
+    /**
+     * Disassociates this content with the specified owner. If the given owner is not associated
+     * with this content, the request is silently ignored.
+     *
+     * @param owner
+     *  The owner to disassociate from this content
+     *
+     * @return
+     *  True if the content was disassociated successfully; false otherwise
+     */
+    public boolean removeOwner(Owner owner) {
+        return owner != null ? this.owners.remove(owner) : false;
     }
 
-    public Owner getOwner() {
-        return this.owner;
+    /**
+     * Sets the owners with which this content is associated.
+     *
+     * @param owners
+     *  A collection of owners to be associated with this content
+     *
+     * @return
+     *  A reference to this content
+     */
+    public Content setOwners(Collection<Owner> owners) {
+        this.owners.clear();
+        if (owners != null) {
```

Comment:
The first line clears the owners collection on the object, the second line checks the input collection.